ALM

```diff
@@ -314,15 +396,21 @@ Public Function RemoveAttachmentFromTest(ByVal TestID, ByVal Filename)
 	If SuccessFlag Then
 		objAttch.RemoveItem(objTempAtch.ID)
 	End If
-
+    ' BUG below should be using RemoveAttachmentFromTest '
 	RenameAttachmentInTest = SuccessFlag
```

Comment:
I havent' looked at vbs for so long I don't feel like I can review this, but removing a reference to a bug to another file doesn't feel like a good answer to me....   Can we just fix this?